flocker/dockerplugin/testtools.py

```diff
@@ -0,0 +1,34 @@
+# Copyright ClusterHQ Ltd.  See LICENSE file for details.
+
+"""
+Testing utilities for ``flocker.dockerplugin`.
+"""
+
+from bitmath import TiB, GiB, MiB, KiB
+
+
+def parse_num(expression):
```

Comment:
Looking at that document, it doesn't appear to suggest a rigid interface that needs to be supported exactly unchanged. 

A further consideration is that it is much easier to compatibility accept more stuff than less stuff. That is, if something was originally an error, than making it valid is a compatible change. But if it was previously valid, making it an error is an incompatible change.